### ISSUE TITLE
What does this mod do?

### ISSUE THREAD

**Issue by owosoup** [Steveplays28/noisium#1]
I'm pretty curious about this mod, and its performance gains, what does it do (in layman's terms)?

I've seen your explanation of replacing functions to optimize blockstate stuff, so I guess it's about filling in the chunk info faster, and not computing the structures themselves?

It's a new mod, so it's unpopular, and I love testing out new work! Especially ones uncalled for, so your project has me really curious :D. Do you have future plans for other optimizations?

**Comment by wicivo** [comment]
> I'm pretty curious about this mod, and its performance gains, what does it do (in layman's terms)?
> 
> I've seen your explanation of replacing functions to optimize blockstate stuff, so I guess it's about filling in the chunk info faster, and not computing the structures themselves?
> 
> It's a new mod, so it's unpopular, and I love testing out new work! Especially ones uncalled for, so your project has me really curious :D. Do you have future plans for other optimizations?

I agree with you, and also is there any sense to use this mod together with c2me
